Hunter/Assets/Scripts/Model/CollisionDetection.cs

```diff
@@ -5,9 +5,9 @@ namespace Hunter.Model
     public static class CollisionDetection
     {
         public static bool AreColliding(Entity firstEntity,
-            Entity secondEntity)
+            Entity secondEntity, float firstRadius, float secondRadius)
         {
-            var radius = firstEntity.BodyRadius + secondEntity.BodyRadius;
+            var radius = firstRadius + secondRadius;
```

Comment:
Secondly - changed collision detection - now it requires more parametrs (sad face), but became more flexible, so We can check collision for everything circular